input/intro-notes/StructureDefinition-au-core-patient-notes.md

```diff
@@ -8,10 +8,15 @@
     <th>Requirements (when used alone or in combination)</th>
   </tr>
   <tr>
-        <td>_id</td>
```

Comment:
This approach doesn't work. The text in this documentation row are the requirements on the single or combination use of the parameter and are the documentation for each of the parameters in the capability statements.

Be careful that whatever you do, may introduce a style that has to be flowed on to every single parameter. 

Workshop a different approach.

Reply:
Table change reverted.
Note included under _id in mandatory parameters.
Project backlog task created to workshop approach: https://github.com/orgs/hl7au/projects/8/views/1?pane=issue&itemId=80511064